### PR TITLE
allow setting background color

### PR DESCRIPTION
**Pull request by esheldon**
Addresses issue #121 

Added ability to set the default background color with

```
[default]

bgcolor = black
```
Also the fg/bg color can be overridden for the screen renderer.
```
[screen]

color = white
bgcolor = black
```

### CODE REVIEW

File: biggles/libplot/renderer.py [hunk 1]
```diff
@@ -98,27 +98,36 @@ def _hexcolor(hextriplet, scale=1):
...
-    if type(color) == type(''):
-        pl.set_colorname_fg(color)
-    else:
+    try:
         r, g, b = _hexcolor(color)
         pl.set_color_fg(r, g, b)
+    except:
+        pl.set_colorname_fg(color)
 
+# this doesn't seem to do anything
```
Comment (beckermr) [inline biggles/libplot/renderer.py]: Yeah that is weird...

**Comment by esheldon** [comment]
According to the docs, you can't set the background color on PS device but I didn't actually check.

persistent was effectively deprecated a while ago, I raised an exception in the code if it was requested.  It never worked.

**Comment by esheldon** [comment]
I checked, you can't set the bg color for PS.  That is odd, since you can in IDL.

**Comment by beckermr** [comment]
Hmmm ok. Well I had some trouble using the write function with background colors I guess for this reason.

**Comment by esheldon** [comment]
It works for `show` and `write_img`

**Comment by esheldon** [comment]
If we move to something like cairo, we can probably make this also work for `write` for png, and possibly pdf.

I was motivated to do this because I think it is nice to have black backgound in some cases, such as for talks and for my own interactive work.  But it isn't so useful for talks if it is not anti-aliased :(  More reason to think of moving to a new backend.

**Pull request merged**